arm-apimanagement/2016-07-07/swagger/apimmanagement.json

```diff
@@ -4453,6 +4453,259 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ApiManagement/service/{serviceName}/backends": {
+      "get": {
+        "tags": [
+          "Backends"
+        ],
+        "operationId": "Backends_ListByService",
+        "description": "Lists a collection of backends in the specified service instance.",
+        "externalDocs": {
+          "url": "https://msdn.microsoft.com/en-us/library/dn935030.aspx"
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ServiceNameParameter"
+          },
+          {
+            "name": "$filter",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "| Field | Supported operators    | Supported functions                         ||-------|------------------------|---------------------------------------------|| id    | ge, le, eq, ne, gt, lt | substringof, contains, startswith, endswith || host  | ge, le, eq, ne, gt, lt | substringof, contains, startswith, endswith |"
```

Comment:
Perhaps insert some line endings so that we can support the tables when GFM is supported